### ISSUE TITLE
Launch Error: orange-canvas not found after install of orange-app

### ISSUE THREAD

**Issue by ahmadia** [ContinuumIO/anaconda-issues#597]
From https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/NP_A7v-1KM4

> Installed orange-app from Anaconda Launcher 1.0.0 Conda 3.19.0 with Python 2.7.11 on win-64. First time launch got Launcher Error "executable not found: orange-canvas". Did conda list and search for "orange" and got orange and orange-app present but no orange-canvas. Did conda install of orange-canvas but no packages found in win-64 channels for it. So, what do I do next? Any suggestions for a conda newbie?

I haven't been able to independently confirm, but it looks like there are problems with the http://orange.biolab.si/ package on Windows.

There appears to be a package here: https://anaconda.org/anaconda/orange

Is orange-canvas a dependency that isn't installed by the above installer?

**Comment by ahmadia** [comment]
Can somebody familiar with Orange please try the install step here:

```
conda install -c https://conda.anaconda.org/anaconda orange
```

And report back if that's working?

**Comment by Hackathorn** [comment]
Thanks! Amazing how you all pounced on this issue... Will standby on suggestions for testing issue resolution.

**Comment by ales-erjavec** [comment]
> Can somebody familiar with Orange please try the install step here:
>     conda install -c https://conda.anaconda.org/anaconda orange
> And report back if that's working?

Running
```
conda install -c https://conda.anaconda.org/anaconda orange
````
does install orange apart from the missing `orange-canvas` executable. That executable is just a entry point for `Orange.OrangeCanvas.main`, and invoking `python -m Orange.OrangeCanvas.main` from the command line does start the application.

Following the conda 'Building packages' tutorial I was able to create this recipe: https://gist.github.com/ales-erjavec/e139c5e4061df90056e9#file-meta-yaml (the `conda skeleton pypi orange` automatically found and added the `orange-canvas = Orange.OrangeCanvas.main:main` entry point to the recipe). Using this (and a functioning mscv compiler) I was able to run `conda build orange` and `conda install --use-local orange`. This did install the executable.
